typings/lhr-lite.d.ts

```diff
@@ -0,0 +1,116 @@
+    export module ResultLite {
+      export interface Category {
+        /** The human-friendly name of the category. */
+        title: string;
+        /** A description of what this category is about (e.g. these help you validate your PWA). */
+        description: string;
+        /** The overall score of the category, the weighted average of all its audits. */
+        score: number;
+        /** An array of references to all the audit members of this category. */
+        auditRefs: AuditRef[];
+        /** An optional description for manual audits within this category. */
+        manualDescription?: string;
+      }
+
+      /**
+       * A reference to an audit result, with weighting and grouping information
+       * for its place in this category.
+       */
+      export interface AuditRef {
+        /** Matches a key in the top-level `audits` object. */
+        auditId: string;
+        /** The weight of the audit's score in the overall category score. */
+        weight: number;
+      }
+
+      export interface Audit {
+        /** The brief description of the audit. The text can change depending on if the audit passed or failed. */
+        title: string;
+        /** A more detailed description that describes why the audit is important and links to Lighthouse documentation on the audit; markdown links supported. */
+        description: string;
+        /** The scored value determined by the audit, in the range `0-1`, or null if `scoreDisplayMode` indicates not scored. */
+        score: number | null;
+        /**
+         * A string identifying how the score should be interpreted:
+         * 'binary': pass/fail audit (0 and 1 are only possible scores).
+         * 'numeric': scores of 0-1 (map to displayed scores of 0-100).
+         * 'informative': the audit is an FYI only, and can't be interpreted as pass/fail. Score is null and should be ignored.
+         * 'notApplicable': the audit turned out to not apply to the page. Score is null and should be ignored.
+         * 'manual': The audit exists only to tell you to review something yourself. Score is null and should be ignored.
+         * 'error': There was an error while running the object (check `errorMessage` for details). Score is null and should be ignored.
+         */
+        scoreDisplayMode: 'binary' | 'numeric' | 'informative' | 'notApplicable' | 'manual' | 'error';
+        /** An explanation of audit-related issues encountered on the test page. */
+        explanation?: string;
+        /** Extra information provided by some types of audits. */
+        details?: Audit.MetricDetails | Audit.OpportunityDetails;
+        /** Error message from any exception thrown while running this audit. */
+        errorMessage?: string;
+      }
+
+      export module Audit {
+        export interface MetricDetails {
+          type: 'metric';
+          /** The value of the metric expressed in milliseconds. */
+          timespanMs?: number;
```

Comment:
I'm still 😬, but I think we mostly exhausted options. last-ditch effort anyone like `ms` `timeMs` `timingMs` more?

Reply:
i consider timespanMs/ms/timingMs to be equally 😬  and i'd be fine with any of those.